src/libraries/Native/Unix/System.Native/pal_networking.c

```diff
@@ -601,12 +601,6 @@ int32_t SystemNative_GetHostEntryForNameAsync(const uint8_t* address, int32_t ad
         return GetAddrInfoErrorFlags_EAI_BADARG;
     }
 
-    sa_family_t platformFamily;
-    if (!TryConvertAddressFamilyPalToPlatform(addressFamily, &platformFamily))
-    {
-        return GetAddrInfoErrorFlags_EAI_FAMILY;
-    }
```

Comment:
What makes this dead?

Reply:
It seems like the TrySetAddressFamily() bellow does it inside so it seems like duplication to me.

Reply:
Ah, not that it's dead code but that it's unnecessary because the same check is performed immediately after.  Ok.

Reply:
Now https://github.com/dotnet/runtime/blob/311de302de8ac21837e98eb359595e733af38c83/src/libraries/Native/Unix/System.Native/pal_networking.c#L617 sets the address family (after the code for sync and async got unified).

The removed code is a left-over from before that unification. The `platformFamily` isn't used anymore, so it's "dead".
This was forgotten in the other PR (due the work in single file host build).

(For completeness...late reply due being in different timezone 😉).